cli/cmd/bind.go

```diff
@@ -0,0 +1,119 @@
+/*
+Copyright 2021 SUSE
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"time"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/tools/clientcmd"
+
+	client "github.com/SUSE/metabroker/apis/generated/clientset/versioned"
+	servicebrokerv1alpha1 "github.com/SUSE/metabroker/apis/servicebroker/v1alpha1"
+)
+
+// NewDefaultBindCmd creates a new bind command with the default dependencies.
+func NewDefaultBindCmd() *cobra.Command {
+	return NewBindCmd(os.Stdout)
+}
+
+// NewBindCmd creates a new bind command.
+func NewBindCmd(
+	stdout io.Writer,
+) *cobra.Command {
+	var credentialNamespace string
+	var instanceNamespace string
+	var secretName string
+	var timeout string
+
+	cmd := &cobra.Command{
+		Use:  "bind [INSTANCE NAME] [CREDENTIAL NAME]",
+		Args: cobra.ExactArgs(2),
+		// TODO: a better description.
+		Short: "Creates a Credential object.",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			instanceName := args[0]
```

Comment:
Do we need to do any validation? Or are we relying on the server for that?

Reply:
Relying on the server.